src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/RequireThisCheck.java

```diff
@@ -684,7 +645,7 @@ private static boolean isUserDefinedArrangementOfThis(AbstractFrame currentFrame
             getAllTokensWhichAreEqualToCurrent(definitionToken, ident,
                 blockEndToken.getLineNo());
 
-        for (DetailAST variableUsage : variableUsagesInsideBlock) {
+        for (DetailAST variableUsage: variableUsagesInsideBlock) {
```

Comment:
wow, no need to do it here. I guess auto-replace is doing wrong thing

Reply:
that's exactly it. I should have been more careful, sorry.